python/interpret_text/experimental/introspective_rationale/introspective_rationale_explainer.py

```diff
@@ -185,46 +185,47 @@ def _load_defaults_model_config_dict(self) -> Dict:
             config = {}
         return config
 
-    def explain_local(self, text: str, **kwargs) -> _create_local_explanation:
+    def explain_local(self, X, y=None, name=None) -> LocalExplanation:
         """ Create a local explanation for a given text
-        :param text: A segment of text
-        :type text: str
-        :param kwargs:
-                preprocessor: an intialized preprocessor to tokenize the
```

Comment:
it seems like it was useful to be able to pass in a preprocessor, can we either:
1. keep this on explain local or
2. put it on the init method

Reply:
Preprocessor made no sense here.

preprocessor has to be set beforehand, and in all of the notebook usages was set. If set, self.preprocessor has the correct one. I think this is leftover code from before the self.preprocessor paradigm was introduced. 

I saw in the usage:

explainer.set_preprocessor(preprocessor)
explainer.explain_local(..., preprocessor=preprocessor)  # self.preprocessor is the same object

Reply:
I see, so for the test preprocessor to be used instead of the training one?

Reply:
can explainer.set_preprocessor() # Serve the same purpose? It looks like it does